people-and-planet-ai/geospatial-classification/trainer/task.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import argparse
+
```

Comment:
This would be a good place to have a high level overview of what the sample does. For people who bump into this code, it might be nice to have a short high level description of what we're trying to achieve and how the data looks like.

Maybe something like [this](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/d8d4951911029b4776bd8d34ad5a8832fe21787b/dataflow/gpu-examples/tensorflow-landsat/main.py#L17-L48)?